src/core/storage/storage-manager.ts

```diff
@@ -233,6 +233,15 @@ export async function initializeStorage(config: StorageConfig): Promise<StorageB
     return await defaultStorageManager.connect();
```

Comment:
nit: we can also get rid of `defaultStorageManager` and the functions that are using it completely